frontend/src/constants/logicConstants.js

```diff
@@ -0,0 +1,1141 @@
+            enabled_continuous: false,
+            enabled_dichotomous: false,
+            enabled_nested: false,
+            threshold: null, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.HIGH_BMD]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 1, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.HIGH_BMDL]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 1, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.LOW_BMD_WARN]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 3, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.LOW_BMDL_WARN]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 3, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.LOW_BMD_FAIL]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 10, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+        [RULE_NAMES.LOW_BMDL_FAIL]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 10, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+        [RULE_NAMES.CONTROL_RESIDUAL_HIGH]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 2, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.CONTROL_STDEV_FIT]: {
+            enabled_continuous: true,
+            enabled_dichotomous: false,
+            enabled_nested: false,
+            threshold: 1.5, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.DOF_ZERO]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: null, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+    },
+};
+
+const headers = [
+    "Test Description",
+    "Continuous",
+    "Dichotomous",
+    "Nested",
+    "Test Threshold",
+    "Bin Placement if Test if Failed",
+    "Notes to Show",
+];
+
+const decision_logic = {
+    recommend_viable: "Recommend model in viable Bin",
+    recommend_questionable: "Recommended model in Questionable Bin",
+    sufficiently_close_bmdl:
+        'BMDL range deemed "sufficiently close" to use lowest AIC instead of lowest BMDL in viable models',
+};
+
+const list = [
```

Comment:
can this be refactored to not use this list? i was thinking you could use the logic.rules array and just render each row in that array as a component. You may need to have some sort of lookup to find the disabled ones, but that would be a easy constant.   For example you could check if bmd_missing + continuous is in the disabled, and if so, disable. That's the only additional lookup you'd need and then you can delete this long list prop

Reply:
you may also need the long-name for a `bmd_missing` lookup